Lesson5/Task5.cs

```diff
@@ -0,0 +1,38 @@
+namespace CSharpFundamentals.Lesson5;
+
+public class Task5
+{
+    public void StringTests()
+    {
+        Console.Write("Enter string: ");
+        string str = Console.ReadLine();
+
+        Console.Write("Entered string: ");
+        Print(str);
+
+        Console.WriteLine();
+
+        int count = CountOfWordsString(str);
+        Console.Write("Count of the words in string: " + count);
+    }
+
+    void Print(string str)
+    {
+        Console.Write(str);
+    }
+
+    int CountOfWordsString(string str)
+    {
```

Comment:
Check your string is null or blank in here

Reply:
If string is not null or blank you must trim it, because if string has a whitespace in string's start position you get wrong count of words.